### DISCUSSION TITLE
Hi guys I recently created an issue 2685 [Jul 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2842] [Discord Threads]
> 🙋‍♂️ **Arascane**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### @seokjun, my bad. will update contributor section. i had missed it out  
#  
> 🙋‍♂️ **Arascane**  
#### <@857998597474484295> Thanks for your prompt reply! Hope it could be reflected in the next patch(not sure when I can check it) 👍  
#  
> 👨‍🦲 **dstala**  
#### release notes updated https://github.com/nocodb/nocodb/releases/tag/0.92.4  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/998060973728481290)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
